build.sbt

```diff
@@ -81,11 +81,25 @@ lazy val dockerSettingsDistroless = Seq(
   Docker / daemonUserUid := None,
   Docker / defaultLinuxInstallLocation := "/opt/snowplow",
   dockerEntrypoint := Seq(
-    "java",
+    "/usr/lib/jvm/java-11-openjdk-amd64/bin/java",
     "-jar",
     s"/opt/snowplow/lib/${(packageJavaLauncherJar / artifactPath).value.getName}"
   ),
-  dockerPermissionStrategy := DockerPermissionStrategy.CopyChown
+  dockerPermissionStrategy := DockerPermissionStrategy.CopyChown,
+
+  Docker / mappings := (Docker / mappings).value ++ Seq(new File("docker/install.sh") -> "install.sh"),
+
+  Docker / dockerCommands := {
+    Seq(
+      Cmd("FROM", "debian:bullseye-slim", "AS", "bullseye"), // Provides standard linux executables for the install script
+      Cmd("COPY", "install.sh", "/usr/bin/"),
+      Cmd("FROM", dockerBaseImage.value, "AS", "java"), // Provides a java installation
+      Cmd("FROM", "gcr.io/distroless/base:nonroot"), // The true base image
```

Comment:
I'm surprised that we can have several `FROM` like this. What does this do exactly ?

Reply:
This documentation describes it better than I can! https://docs.docker.com/build/building/multi-stage/

Reply:
Interesting, thanks for sharing!